### DISCUSSION TITLE
Using a restclient in a Keycloak 23 extension

### DISCUSSION THREAD

**Discussion by koplas** [keycloak/keycloak#25612] [Q&A]
With the release of Keycloak 23, it is no longer possible to use the `ResteasyClientBuilder`.

Are there other rest clients that could be used inside a keycloak extension?
I tried `quarkus-resteasy-reactive` without success as `kc.sh build` fails with: `io.smallrye.config.ConfigSourceFactory: io.smallrye.config.PropertiesLocationConfigSourceFactory not a subtype` if I include it as a dependency.

Using RestEasy results in dependency problems when building with quarkus. The workaround described in https://github.com/keycloak/keycloak/discussions/16241 does not work for me.

**Comment by DoboszKacper** [comment]
@koplas did you manage to resolve this issue?

**Comment by koplas** [comment]
I described a workaround here: https://github.com/keycloak/keycloak/issues/25589#issuecomment-1868517798.
Instead of `keycloak-admin-client-reactive` and `keycloak-admin-client-reactive-deployment`, you add `quarkus-resteasy-reactive` and `quarkus-resteasy-reactive-deployment`.

**Reply by xgp** [reply]
FYI this workaround no longer works as of `26.2.0`. Strangely, it causes compilation errors in `quarkus/runtime`.
